packages/contractkit/src/wrappers/Validators.ts

```diff
@@ -164,13 +169,20 @@ export class ValidatorsWrapper extends BaseWrapper<Validators> {
     const res = await this.contract.methods.getValidator(address).call()
     return {
       address,
-      ecdsaPublicKey: res[0] as any,
-      blsPublicKey: res[1] as any,
-      affiliation: res[2],
-      score: fromFixed(new BigNumber(res[3])),
+      // @ts-ignore incorrect typing for bytes
+      ecdsaPublicKey: res.ecdsaPublicKey,
```

Comment:
there's a "parser" on BaseWrapper that "hides" this type problem. parseBytes or similar